api_blog/views.py

```diff
@@ -1,11 +1,60 @@
+import json
+
 import sqlalchemy as sa
 from aiohttp import web
+from sqlalchemy.future import select
 
-from .models import Session
+from .models import Session, User
+from .security import generate_password_hash
 
 
 async def index(request):
     async with Session() as session:
         async with session.begin():
             await session.execute(sa.select(1))
     return web.Response(text="hello aiohttp")
+
+
+async def sign_up(request):
```

Comment:
Let's use cbv here. It's better to use fbv for simple stuff, such as health check or webhooks